backend/python/migrations/versions/dedc5f7be10c_initial_migration.py

```diff
@@ -36,7 +36,11 @@ def upgrade():
         sa.Column("first_name", sa.String(), nullable=False),
         sa.Column("last_name", sa.String(), nullable=False),
         sa.Column("auth_id", sa.String(), nullable=False),
-        sa.Column("role", sa.Enum("User", "Admin", name="roles"), nullable=True),
+        sa.Column(
+            "role",
+            sa.Enum("Admin", "Regular Staff", "Relief Staff", name="roles"),
+            nullable=True,
```

Comment:
Do we want this to be nullable? Or should we require every user we create has a role

Reply:
should be false, updated

Reply:
I think u might have forgotten to push the change for this.

Reply:
this is for the initial migration, on the new migration they should be false

Reply:
u right